src/app/shared/contextHelper.ts

```diff
@@ -1,13 +1,11 @@
 import { PHONE_TYPES } from "../../constants/device-constants";
 import TechnicalError from "../../errors/technical-error";
 
-export function validatePhoneType(context?: string): asserts context is string {
-  if (
-    !context ||
-    !([PHONE_TYPES.IPHONE, PHONE_TYPES.ANDROID] as string[]).includes(context)
-  ) {
-    throw new TechnicalError(
-      `Context cannot be parsed as a phone type: ${context}`,
-    );
-  }
+export function validatePhoneType(context?: string): void {
```

Comment:
It might be cleaner/clearer if we just make this a function that returns a `PHONE_TYPES` given context?

i.e. if context.startsWith("android") -> `PHONE_TYPES.ANDROID`

then `getAppStoreRedirectUrl` can still use an actual `PHONE_TYPES` rather than a context string pretending to be one